python/lsst/meas/astrom/astrom.py

```diff
@@ -254,11 +259,11 @@ def useKnownWcs(self, sources, wcs=None, exposure=None, filterName=None, imageSi
         probably also want to turn OFF 'calculateSip'.
 
         This involves searching for reference sources within the WCS
-        area, and matching them to the given 'sources'.  If
+        area, and matching them to the given 'sourceCat'.  If
```

Comment:
I think it'd have been best to put all these trivial name changes in code you're not otherwise modifying in a separate commit.  I don't suppose it'd still be possible to separate them out?